### PR TITLE
Update 2021

### PR DESCRIPTION
**Pull request by sakrifor**
Update for 2021 data. @GregB712 can you please take a look?

### CODE REVIEW

File: TrainModel.ipynb [hunk 1]
```diff
@@ -45,18 +43,97 @@
...
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "ects = pd.read_csv(\"subjects_ects.csv\")\n",
+    "ects_dict = ects.set_index('Subjects')['Ects'].to_dict()\n",
+    "#display(ects_dict)"
```
Comment (GregB712) [inline TrainModel.ipynb]: Is this leftover code from something else, because I couldn't find any usage.

File: TrainModel.ipynb [hunk 2]
```diff
@@ -45,18 +43,97 @@
...
+    }
+   },
+   "outputs": [],
+   "source": [
+    "#data.describe()\n",
+    "#data.info()"
    ]
   },
   {
    "cell_type": "markdown",
```
Comment (GregB712) [inline TrainModel.ipynb]: Even though its comments, maybe the cell type here should be "code"?

File: TrainModel.ipynb [hunk 3]
```diff
@@ -45,18 +43,97 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
    "execution_count": null,
-   "outputs": []
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Change Courses Names\n",
```
Comment (GregB712) [inline TrainModel.ipynb:76]: You can find those function in `Parser.ipynb` too. The final csv file, that we extract from the parser, has already the correct names.

**Comment by sakrifor** [comment]
@GregB712 If everything is okay you can squash and merge :)

**Pull request merged**